integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -601,8 +606,8 @@ public int release(String path, FuseFileInfo fi) {
     LOG.trace("release({})", path);
     OpenFileEntry oe;
     final long fd = fi.fh.get();
+    oe = mOpenFiles.getFirstByField(ID_INDEX, fd);
```

Comment:
I think we can keep this in the synchronized, otherwise it could be removed already? This call should not be expensive

Reply:
Double `release` for the same file in an Alluxio-Fuse client should not happen. If double `release` then remove will error out and it is expected

Reply:
Double release for the same file in an Alluxio-Fuse client should not happen.

Reply:
but to be safe, we could add it back